packages/@glimmer/util/lib/object-utils.ts

```diff
@@ -31,19 +31,11 @@ function assignFn(obj: any) {
 export let assign = Object.assign ?? assignFn;
 
 export function fillNulls<T>(count: number): T[] {
-  let arr = new Array(count);
-
-  for (let i = 0; i < count; i++) {
-    arr[i] = null;
-  }
-
-  return arr;
+  return new Array(count).fill(null);
 }
 
-export function values<T>(obj: { [s: string]: T }): T[] {
-  const vals = [];
-  for (const key in obj) {
-    vals.push(obj[key]);
-  }
-  return vals;
+export const objectValues = Object.values ?? objectValuesFn;
```

Comment:
Hmm, why can't we assume that `Object.values` is present? I assume that if we drop IE11 we can drop it..

Reply:
we have CI for IE11, and it will fail

Reply:
lol, same with Array.fill

Reply:
Ie 11 support is dropped. Can this be updated? (Does it need to be?)

Reply:
@NullVoxPopuli let me try.